moveit_ros/planning/planning_request_adapter_plugins/src/resolve_constraint_frames.cpp

```diff
@@ -32,52 +32,51 @@
  *  POSSIBILITY OF SUCH DAMAGE.
  *********************************************************************/
 
-/* Author: Robert Haschke */
+/* Author: Robert Haschke
+   Desc: This adapter changes the link_name field of a constraint from an object's (sub-)frame name to the name of the
+   robot link, that object is attached to. Transforming the frame names is necessary because the frames of an attached
+   object are not know to a planner.
```

Comment:
typos

```suggestion
   Desc: This adapter changes the link_name field of a constraint from an object's (sub-)frame name to the name of the
   robot link, that the object is attached to. Transforming the frame names is necessary because the frames of an attached
   object are not known to a planner.